eslint-rules/rules/no-external-imports.ts

```diff
@@ -13,13 +14,14 @@ module.exports = {
     },
     schema: [],
   },
+  defaultOptions: [],
   create: function (context) {
     if (!context.filename.includes("text-mode-core")) return {};
     return {
       ImportDeclaration: function (node) {
-        if (node.source.type === "Literal" && node.importKind !== "type") {
+        if (node.importKind !== "type") {
```

Comment:
Surprised the `source` is typed only as `Literal` and not `Literal | Identifier`, ref [estree ImportDeclaration spec](https://github.com/estree/estree/blob/35aa0852c62696cf578f65caf129313c789c8057/experimental/module-declarations.md?plain=1#L32).

Removal looks fine I guess.